runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -99,12 +116,40 @@ protected void shutDown() throws Exception
 		overlayManager.remove(overlay);
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.BA_TEAM_INFO_GROUP_ID)
+		{
+			if (new WorldPoint(2579, 5295, 0).distanceTo(client.getLocalPlayer().getWorldLocation()) < 5)
```

Comment:
Use constant for this world point, also just use startedFromWaveOne = worldPoint.distanceTo(client.getLocalPlayer().getWorldLocation()) < 5